### PR TITLE
Community 1.2 doc updates pointfix

### PR DESCRIPTION
**Pull request by ganesaar**

### CODE REVIEW

File: docs/User/SWCI_APIs.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ The python3 program needs to import the SWCI class, and then use the below APIs.
 |`listTasks()`|This method displays the list of tasks that are registered into the Swarm Learning network.| |
 |`listTrainingContracts()`|This method displays the list of training contracts registered with Swarm Learning network.| |
 |`ls()`|This method displays the directory contents of the SWCI container.|`optStr=''`|
-|`plotTopology()`|This method displays the PNG object showing the current topology of the Swarm Learning network.| |
+|`plotTopology()`|This method displays the PNG object showing the current topology of the Swarm Learning network.<br> By default, it will display the Swarm Node type and Host IP on which the node is running.<br> User can pass additional attributes which they want to see on the plot. These additional attributes are: 'Port', 'ContainerName', 'UUID', 'parentUUID' and 'i-am-alive'.<br> **NOTE**: Color codes are hexadecimal triplets representing the colors red, green, and blue (#RRGGBB).|SNColour="#ADD8E6",<br> SWOPColor="#33FF33",<br> SLColor="#FFCCCB", <br> attrs=['ContainerName']|
```
Comment (suresh-ls) [inline docs/User/SWCI_APIs.md]: Make it 2 para's.

new-line. 

"User can pass additional attributes which they want to..."

File: docs/User/SWCI_APIs.md [hunk 2]
```diff
@@ -38,7 +38,7 @@ The python3 program needs to import the SWCI class, and then use the below APIs.
 |`listTasks()`|This method displays the list of tasks that are registered into the Swarm Learning network.| |
```
Comment (suresh-ls) [inline docs/User/SWCI_APIs.md:38]: For listNodes() - This method displays the list of Swarm nodes that have registered and are currently active. 
For each Swarm node, it displays the Node type, Host IP, Port, Container name, UUID, parent UUID and the last received ‘i-amalive’ Timestamp.
 
SWCI and ML nodes are not displayed.

Comment (suresh-ls) [inline docs/User/SWCI_APIs.md:38]: For assignTask() - 

Assigns a task to a taskrunner instance and specify the minimum peer count for declaring the result. This triggers SWOPs to execute this task

<br>- `taskName`: Specifies the unique name of task.<br>- `trName`: Specifies the unique name of Taskrunner. SWOP’s listening on this Taskrunner participates and executes this task.<br>- `WITH <peersNeeded> PEERS`:<br>`peersNeeded` is a nonzero positive integer specifying the minimum number of peers required to complete this task.<br>

Meaningful value of minimum peers is dependent on the task type. For RUN\_SWARM task, it is the number of SL and ML node pairs. For all other TASK types, it is the number of SWOP nodes.<br>- For RUN_SWARM task type, the actual numbers of SL/ML peers started could be equal or greater than `peersNeeded`, depending on the number of the SL nodes defined in the SWOP profiles.<br>- For other TASK types, the number of SWOPs participating would be equal to `peersNeeded`

File: docs/User/SWCI_APIs.md [hunk 3]
```diff
@@ -38,7 +38,7 @@ The python3 program needs to import the SWCI class, and then use the below APIs.
...
 |`listTrainingContracts()`|This method displays the list of training contracts registered with Swarm Learning network.| |
```
Comment (suresh-ls) [inline docs/User/SWCI_APIs.md:39]: Need to add:

`getTaskrunnerStatus() <trName:string>`

|Displays the current status of the specified Taskrunner.<br> `trName`: Specifies the unique name of Taskrunner.<br> Provides below information like,<br>- TASK NAME – Current running task or ‘Empty’ if no task is assigned.<br>- PEER TYPE – SL for RUN_SWARM tasks, SWOP for all other tasks.<br>- TASK STATE – Current task state.<br>- ACTIVE, COMPLETED and FAILED PEERS information.<br>- TIME STAMPs of various events on the TASKRUNNER contract.

2) createContext()  needs to reflect the new argument in the corresponding SWCI command

**Review by suresh-ls — COMMENTED** [review]
Where are the AMD changes ?

File: docs/User/Swarm_Learning_Command_Interface.md [hunk 4]
```diff
@@ -65,19 +66,22 @@ A SWCI tool can control operations on multiple Swarm Learning framework instance
...
+|`GET TASKRUNNER PEER STATUS <taskRunnerName:string><swopIndex\swopUid>:`|Displays current status for the specified PEER in the Taskrunner’s context.<br>-   `taskRunnerName`: Specifies the unique name of Taskrunner.<br>-   `swopIndex`: The index number of SWOP Node, starts with 0 up to ENROLLEDSWOP.<br>-   `swopUid:`: The UID String of the SWOP Node. You can specify either `swopIndex` or `swopUid`, not both.<br>The status of the PEER differs based on the type of the current TASK that has been assigned to TASKRUNNER.<br>  For RUN_SWARM task, the status summary reports SWOP node UID, Number of SL PEERs this SWOP has spawned, and list of all SL node information (UID, Status, Description).<br> For all other types of tasks, the status summary reports SWOP node status (UID, Status, Description).<br> **Note**: Node UID can be used to identify the container name/id from ‘LIST NODES’ command. With container name/id, user can debug the error with docker logs.|
```
Comment (iArpanPatel) [inline docs/User/Swarm_Learning_Command_Interface.md]: When looked into the preview mode(Display the rich diff), these 2 commands seems to have merged in single row. Add 'break' at appropriate place.

File: docs/User/SWCI_APIs.md [hunk 5]
```diff
@@ -29,16 +29,17 @@ The python3 program needs to import the SWCI class, and then use the below APIs.
 |`getTaskRunnerInfo()`|This method prints the current status of the specified Taskrunner ID.|`trName`|
 |`getTaskRunnerPeerStatus()`|This method prints the current status for the specified PEER (SWOP node) in the Taskrunner’s context.|`trName`, `idx`|
 |`getTaskRunnerStatus()`|This method prints the overall status of the taskrunner.|`trName`|
+|`getTaskrunnerStatus() <trName:string>`|Displays the current status of the specified Taskrunner.<br> `trName`: Specifies the unique name of Taskrunner.<br> Provides below information like,<br>- TASK NAME – Current running task or ‘Empty’ if no task is assigned.<br>- PEER TYPE – SL for RUN_SWARM tasks, SWOP for all other tasks.<br>- TASK STATE – Current task state.<br>- ACTIVE, COMPLETED and FAILED PEERS information.<br>- TIME STAMPs of various events on the TASKRUNNER contract.| |
```
Comment (suresh-ls) [inline docs/User/SWCI_APIs.md]: I see 2 getTaskrunnerStatus()..  Remove the 1st one..

Also all the arguments and its description, can be moved to the 3rd column.

File: docs/User/SWCI_APIs.md [hunk 6]
```diff
@@ -13,10 +13,10 @@ The python3 program needs to import the SWCI class, and then use the below APIs.
...
+|`assignTask()`|This method assigns a task to a instance and specify the minimum peer count for declaring the result.|This triggers SWOPs to execute this task<br>- `taskName`: Specifies the unique name of a task.<br>- `trName`: Specifies the unique name of a Taskrunner. SWOP’s listening on this Taskrunner participates and executes this task.<br>- `WITH <peersNeeded> PEERS:`<br>- `peersNeeded` is a nonzero positive integer specifying the minimum number of peers required to complete this task.<br> Meaningful value of minimum peers is dependent on the task type. For `RUN_SWARM` task, it is the number of SL and ML node pairs. For all other task types, it is the number of SWOP nodes.<br>- For RUN_SWARM task type, the actual numbers of SL/ML peers started could be equal or greater than `peersNeeded`, depending on the number of the SL nodes defined in the SWOP profiles.<br>- For other TASK types, the number of SWOPs participating would be equal to `peersNeeded`.|
```
Comment (suresh-ls) [inline docs/User/SWCI_APIs.md]: task to a instance -> task to a Taskrunner instance

File: docs/User/SWCI_APIs.md (hunk 1 above)
Comment (suresh-ls) [reply docs/User/SWCI_APIs.md]: This para can be moved to 3rd column - "User can pass additional attributes which they want to see on the plot... "

**Review by suresh-ls — APPROVED** [review]

File: docs/Install/Starting_SWOP_nodes.md [hunk 7]
```diff
@@ -6,4 +6,6 @@ Use the `swarm-learning/bin/run-swop` script to start SWOP nodes. This script ac
 |--------------|-----------|-------------|
 |`--usr-dir <dir>`\(Mandatory Parameter\)<br>| Host directory that should be used as the user directory by this SWOP node.<br> | |
 |`--profile-file-name <swop-profile file>`\(Mandatory Parameter\)<br>|This file should be located inside the user directory, at the top-level itself.| |
-
+|`--sn-ip <IP address or DNS name>`<br>|The IP address or DNS name of the host system on which the Swarm Network node with which this Swarm Learning node must associate, while running.|None|
```
Comment (RadhakrishnaJ) [inline docs/Install/Starting_SWOP_nodes.md]: hard to read. can be this simplified?

**Review by RadhakrishnaJ — APPROVED** [review]
Reverse proxy related diagrams and content is not reviewed under this.

File: docs/User/Frequently_asked_questions.md [hunk 8]
```diff
@@ -64,6 +64,34 @@ If you are using SWOP to launch concurrent training, you need to have separate S
 
 SWCI is designed to work with several swarm networks at once. Therefore, you can create a context and switch to that context to execute commands. Each context identifies which SN the SWCI must connect to.
 
+## What are the supported SWCI commands?
+
```
Comment (RadhakrishnaJ) [inline docs/User/Frequently_asked_questions.md:88]: What about AMD FAQs?

**Review by RadhakrishnaJ — APPROVED** [review]
Except reverse proxy reviewed rest of the content.

File: docs/User/SWCI_APIs.md [hunk 9]
```diff
@@ -28,17 +28,17 @@ The python3 program needs to import the SWCI class, and then use the below APIs.
...
 |`listTaskRunners()`|This method displays the list of taskrunners that are registered into the Swarm Learning network.| |
 |`listTasks()`|This method displays the list of tasks that are registered into the Swarm Learning network.| |
 |`listTrainingContracts()`|This method displays the list of training contracts registered with Swarm Learning network.| |
 |`ls()`|This method displays the directory contents of the SWCI container.|`optStr=''`|
-|`plotTopology()`|This method displays the PNG object showing the current topology of the Swarm Learning network.| |
+|`plotTopology()`|This method displays the PNG object showing the current topology of the Swarm Learning network.<br> By default, it will display the Swarm Node type and Host IP on which the node is running.<br> **NOTE**: Color codes are hexadecimal triplets representing the colors red, green, and blue (#RRGGBB).|User can pass additional attributes which they want to see on the plot. These additional attributes are: 'Port', 'ContainerName', 'UUID', 'parentUUID' and 'i-am-alive'.<br> </br> SNColour="#ADD8E6",<br> SWOPColor="#33FF33",<br> SLColor="#FFCCCB", <br> attrs=['ContainerName']|
```
Comment (cdivyanambiar) [inline docs/User/SWCI_APIs.md:41]: RGB is good enogh

File: docs/Install/Exposed_port_numbers.md [hunk 10]
```diff
@@ -13,7 +13,8 @@ Depending on the type of Swarm Learning components that are running on a host, s
 -   \(Optional\). An SWCI API server port that is used by the SWCI node to run a REST based API service. By default, port 30306 is used.
 
 <blockquote>
-    NOTE: If you use different ports other than the default port, you must open those ports accordingly. For instance, in our MNIST example, we are using ports     16000 and 18000 for the SL File server ports, which must be opened.
+    NOTE: If you use different ports other than the default port, you must open those ports accordingly. For instance, in our MNIST example, we are using ports     16000 and 18000 for the SL File server ports, which must be opened.<br>
```
Comment (cdivyanambiar) [inline docs/Install/Exposed_port_numbers.md]: Extra space after port

**Pull request merged**